package.json

```diff
@@ -13,9 +13,9 @@
     "format": "npm run lint -- --fix",
     "build": "grunt",
     "watch": "grunt watch",
-    "package": "grunt package",
+    "package": "npm run build && node ./scripts/package.js",
```

Comment:
Can we change this script's name to contain `electron`. 
As it stands, it can be confusing with `npm pack`, which creates the archive that is used by npm.